scripts/kalibro_configurations/recipe.rb

```diff
@@ -33,7 +33,8 @@ def build
       s.gsub! /^(\s*)password:(.*)/, ''
     end
 
-    generate_post_install("#{File.dirname(__FILE__)}/post_install.sh", 'kalibro-configurations', 8083)
+    generate_script(workdir("../post_install.sh"), workdir("post_install.sh"), 'kalibro-configurations', 8082)
```

Comment:
The port may be a copy/paste error. Configuretions should get started on port 8083.